babel.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = function(api) {
     case "standalone":
       includeRegeneratorRuntime = true;
       unambiguousSources.push("packages/babel-runtime/regenerator");
+    // fall through
```

Comment:
I hate that prettier doesn't include this comment, since conceptually it's a control flow statement similar to `break`/`continue`.

Reply:
Classic ambiguous comment indentation 😆